rosapi/interface.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+
+class Interface():
+  def __init__(self, api, interface):
+    self.interface_name = interface
+    self.api = api
+
+  def checkinterface(self):
```

Comment:
Who/what are we trying to be consistent with?

CLI is not always consistent with the API.